classification/classification_network.py

```diff
@@ -241,108 +230,132 @@ def createClassifierRegion(network, classifierType, classifierParams, prevRegion
+               srcOutput="resetOut", destInput="resetIn")
 
   # Feed forward link from SP to TM
   network.link("SP", "TM", "UniformLink", "",
-      srcOutput="bottomUpOut", destInput="bottomUpIn")
+               srcOutput="bottomUpOut", destInput="bottomUpIn")
 
   # Feedback links (unnecessary??)
   network.link("TM", "SP", "UniformLink", "",
-      srcOutput="topDownOut", destInput="topDownIn")
+               srcOutput="topDownOut", destInput="topDownIn")
   network.link("TM", "sensor", "UniformLink", "",
-      srcOutput="topDownOut", destInput="temporalTopDownIn")
+               srcOutput="topDownOut", destInput="temporalTopDownIn")
 
   # Forward the sensor region sequence reset to the TM
   network.link("sensor", "TM", "UniformLink", "",
-      srcOutput="resetOut", destInput="resetIn")
+               srcOutput="resetOut", destInput="resetIn")
 
   # Feed the TM states to the classifier.
   network.link("TM", "classifier", "UniformLink", "",
-      srcOutput = "bottomUpOut", destInput = "bottomUpIn")
+               srcOutput="bottomUpOut", destInput="bottomUpIn")
+
+  # Feed the predicted active cells to the classifier
+  network.link("TM", "classifier", "UniformLink", "",
+               srcOutput="predictedActiveCells", destInput="predictedActiveCells")
 
   # Link the sensor to the classifier to send in category labels.
   network.link("sensor", "classifier", "UniformLink", "",
-      srcOutput = "categoryOut", destInput = "categoryIn")
+               srcOutput="categoryOut", destInput="categoryIn")
 
 
-def createNetwork(args):
+def createNetwork(dataSource,
+                  sensorType,
+                  encoders,
+                  numCategories,
+                  classifierType,
+                  classifierParams,
+                  classifyPredictedActiveCells):
   """
   Create the network instance with regions for the sensor, SP, TM, and
   classifier. Before running, be sure to init w/ network.initialize().
 
-  @param args                 (dataSource, sensorType, encoders, numCategories, 
-                                classifierType, classifierParams) , more info:
-    dataSource   (RecordStream) Sensor region reads data from here.
-    sensorType   (str)          Specific type of region, e.g. "py.RecordSensor";
-                                possible options can be found in nupic/regions/.
-    encoders     (dict)         See createEncoder() docstring for format.
-    numCategories  (int)        Max number of categories of the input data.
-    classifierType   (str)      Specific type of classifier region, e.g. "py.SequenceClassifier";
-                                possible options can be found in nupic/regions/.
-    classifierParams   (dict)   Parameters for the model. E.g. {'maxCategoryCount': 3}                               
-                                
-  @return        (Network)      sensor -> SP -> TM -> CLA classifier
+  @param dataSource   (RecordStream) 
```

Comment:
As in `createClassifierRegion()`, the spacing in this docstring makes it difficult to read.